.github/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+<!-- markdownlint-disable first-line-heading -->
+## Ticket(s)
+<!--- Please link the Jira/Linear card here -->
+
+- [QUE-###](https://linear.app/larcity-and-affiliates/issue/QUE-###)
```

Comment:
This project within Linear doesn't exist yet... just including it in our documentation as a placeholder for when we start working with cards in a project management tool to keep track of feature work/bug fixes etc.